services/github/github-all-contributors.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { renderContributorBadge } = require('../contributor-count')
+const { ConditionalGithubAuthV3Service } = require('./github-auth-service')
+const { fetchJsonFromRepo } = require('./github-common-fetch')
+const { documentation } = require('./github-helpers')
+
+const schema = Joi.object({
+  contributors: Joi.array().required(),
+}).required()
+
+module.exports = class GithubAllContributorsService extends ConditionalGithubAuthV3Service {
+  static get category() {
+    return 'activity'
+  }
+
+  static get route() {
+    return {
+      base: 'github/all-contributors',
+      pattern: ':user/:repo',
```

Comment:
Sorry one more question/thought regarding the branch. In the referenced issue you mentioned:

> As the Bot updates the master branch,

Is that always explicitly master, or is it the default branch in the repository?

Reply:
Just done some testing, it does indeed push to the default branch not always master. I will look into adding a way to handle that, and a specified branch.

Reply:
We can just add the `branch` route param, looks like you've already got everything else wired up to handle it :+1: 

```suggestion
      pattern: ':user/:repo/:branch*',
```

Reply:
I've implemented that, as well as update the example given to the Frontend, from the searching I have done it seems there's currently no way to use the default branch (Issue #4114) if a branch isn't specified, would the be correct?

Reply:
Correct. We do the best we can today, and for folks with different defaults they need to include an extra param in their badge url